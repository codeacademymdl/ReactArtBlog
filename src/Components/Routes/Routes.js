import React, { Component } from 'react';
import App from '../../App';
import CreatePostForm from '../CreatePostForm/ CreatePostForm'
import { BrowserRouter as Router, Route } from 'react-router-dom';

class Routes extends Component {
    render() {
        return (
            <Router>
            <div>
                <Route exact path="/" component={App}></Route>
                <Route exact path="/form" component={CreatePostForm}></Route>
            </div>
        </Router>
        );
    }
}

export default Routes;




