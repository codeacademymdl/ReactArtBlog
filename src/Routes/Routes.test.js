import React from 'react';

import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Routes from './Routes'

describe('Routes', () => {
    it('renders correctly', () => {
        const tree = renderer
          .create(<Routes />)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
});

