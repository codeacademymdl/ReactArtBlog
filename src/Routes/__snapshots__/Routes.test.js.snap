// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes renders correctly 1`] = `
<div>
  <div>
    <div
      className="nav"
    >
      <div
        className="heading"
      >
        <h1>
          The Artifact
        </h1>
        <h2>
          Culture & Art Blog
        </h2>
      </div>
      <div
        className="navLink"
      >
        <a
          href="#!"
        >
          Blog
        </a>
        <a
          href="#!"
        >
          About
        </a>
        <a
          href="#!"
        >
          Contact
        </a>
      </div>
    </div>
    <div
      className="card_grid"
    />
    <a
      className="linkBtn"
      href="/form"
      onClick={[Function]}
    >
      <div
        className="createPost"
      >
        <button
          className="button"
        >
          CreatePost
        </button>
      </div>
    </a>
    <footer
      className="footer"
    >
      <p>
        Â©artifact.com 2019
      </p>
    </footer>
  </div>
</div>
`;
